LondonBritishLibrary/orient/BLorient12071A.xml

```diff
@@ -0,0 +1,177 @@
+                        <idno>BL Oriental 12071A</idno>
+                        <altIdentifier><idno>Or. 12071A</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 86</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT7255Secrets"/>
+                            <incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ አብ፡ ዘኢይትገሠሥ፡ እሳት፡ ወበስመ፡ እግዚአብሔር፡ ወልድ፡ ዘያደነግፅ፡ መብርቅ፡ ወበስመ፡ እግዚአብሔር፡
+                                መንፈስ፡ ቅዱስ፡ ዘያውዔ፡ ዕንቈ፡ ባሕርይ፡ ብርሃነ፡ ታዴስ፡ ፯ጊዜ፡ አኔዎስ፡ ፫ጊዜ፡ <gap reason="ellipsis"/> በዝአስማት፡ ክሥት፡ ሊተ፡</incipit>
+                            <explicit xml:lang="gez"><expan><abbr>ለገ</abbr><ex>ብርከ</ex></expan>፡ 
+                                <expan><abbr> እገ</abbr><ex>ሌ</ex></expan>። ቀለምሲስ፡ ዮሐንስ፡ ወንጌል፡ እምጥንቱ፡ እስከ፡ ተፍጻሜቱ፡ በአውጥ፡ ቅፀል፡ ጸሒፈከ፡ 
+                                በዕፍራን፡ ቀለም፡ ለአጽንየ፡ በሐዲስ፡ ቆባ፡ ማር፡ ፯ቀን፡ ድግም፡ በዘኢያገድፍ፡ በጥቁር፡ ገብስ፡ ድፍድፍ፡ ፯ቀን፡ ድቅም።</explicit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT7259Mestir"/>
+                            <note>Possibly the continuation of <ref target="#ms_i1"/>.</note>
+                            <incipit xml:lang="gez">በስመ፡ አብ፡ በል፡ ጸሎት፡ በእንተ፡ ምስጢር፡ አድፍኤል፡ ፯ጊዜ፡ ወጢሙር፡ ፯ጊዜ፡</incipit>
+                            <explicit xml:lang="gez">ክሥት፡ ለገብርከ፡ እገሌ፡ ምስለ፡ ጸሎተ፡ ኤፍሬም፡ ወዓምደ፡ ብርሃን፡ በ፵፱ዘቢብ፡ በስም፡ ጽዋዕ፡ ፯ቀን፡ ድግም፡ ለንባብ፡ በሽንብራ፡ ቂጣ፡
+                                ጽሒፈከ፡ በነጭ፡ ዶሮ፡ መረቅ፡ ፈትፍተሕ፡ አብልዖ፡ ለሕፃን።</explicit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT4005Salota"/>
+                            <note>With the addition of ʾasmāt in the incipit and considerably shorter in the end as in
+                                <ref type="mss" corresp="EMIP02338#a1">Portland, Ethiopic Manuscript Imaging Project, Weiner Codex 306, fol. ir</ref>
+                                and <ref type="mss" corresp="EMIP00447#ms_i12">Portland, Ethiopic Manuscript Imaging Project, Kaleab Addis Project 47, fols 94v-95r </ref>.</note>
+                            <incipit xml:lang="gez">በስመ<supplied reason="omitted">፡</supplied> አብ፡ በል፡ ጸሎተ፡ በረድ፡ ምስማኤል፡ ምምኤል፡</incipit>
+                            <explicit xml:lang="gez">ኢታውርድ፡ በረደ፡ ዘእንበለ፡ ጽሩይ፡ ማይ፡ በሃገረ፡ <expan><abbr> እገ</abbr><ex>ሌ</ex></expan>፡</explicit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <title ref="LIT7260PPKumael"/>
+                            <incipit xml:lang="gez">ኩማኤል፡ ፫ጊዜ፡ ኽኤል፡ ፫ጊዜ፡ ውርዝያል፡ ሠነገ፡ ባሕር፡ ዘትትሜሰል፡ ሰብእ፡ ምስለ፡ ሰብእ፡ ነዓ፡ ኀቤየ፡ መደንግፀ፡ ጋኔን፡ ለአነ፡</incipit>
```

Comment:
You mark when እገሌ is abbreviated, but you ignore when ጊዜ is abbreviated. Why?

Reply:
Ok, it seemed like it was done regularly, so I thought maybe there was a reason that I couldn't understand :) Thank you!